docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay/upgrade-basics/using-the-database-upgrade-tool.md

```diff
@@ -22,6 +22,12 @@ If you're [upgrading to a new Liferay Docker image](../../installing-liferay/usi
 
 1. Set up the [File Store (Document Library)](../../../system-administration/file-storage/configuring-file-storage.md) by copying it from your [backup](../../maintaining-a-liferay-dxp-installation/backing-up.md) to the new installation and or configuring the new installation to use it via a [`.config` file](../../../system-administration/configuring-liferay/understanding-configuration-scope.md).
```

Comment:
change the link to https://help.liferay.com/hc/en-us/articles/360017877292-Understanding-System-Configuration-Files

Mention export/import when using AdvancedFileSystemStore